tests/ui/test_ui_tools.py

```diff
@@ -433,19 +433,17 @@ def test_update_streams(self, mocker, stream_view, new_text, expected_log):
                 ] == expected_log
         self.view.controller.update_screen.assert_called_once_with()
 
-    def test_mouse_event(self, mocker, stream_view):
-        mocker.patch.object(stream_view, 'keypress')
+    @pytest.mark.parametrize('button, key', [
+            (4, 'up'),
+            (5, 'down'),
+        ],
+        ids=['scroll_wheel_up', 'scroll_wheel_down']
+    )
+    def test_mouse_event(self, mocker, stream_view, button, key):
         size = (200, 20)
-        col = 1
-        row = 1
-        focus = "WIDGET"
-        # Left click
-        stream_view.mouse_event(size, "mouse press", 4, col, row, focus)
-        stream_view.keypress.assert_called_once_with(size, "up")
-
-        # Right click
-        stream_view.mouse_event(size, "mouse press", 5, col, row, focus)
-        stream_view.keypress.assert_called_with(size, "down")
+        mocker.patch.object(stream_view, 'keypress')
+        stream_view.mouse_event(size, 'mouse press', button, 1, 1, 'WIDGET')
+        stream_view.keypress.assert_called_once_with(size, key)
```

Comment:
These tests are looking a lot better :+1: 

Would it be logical to introduce the fixture in the next commit in this one? If so, you could focus on one commit introducing and using the new fixture, and put the invalid test changes into another for now?